PowerEditor/src/ScintillaComponent/FindReplaceDlg.cpp

```diff
@@ -5124,15 +5140,6 @@ LRESULT APIENTRY Progress::wndProc(HWND hwnd, UINT umsg, WPARAM wparam, LPARAM l
 			break;
 		}
 
-		case WM_PRINTCLIENT:
-		{
-			if (NppDarkMode::isEnabled())
-			{
-				return TRUE;
-			}
-			break;
-		}
-

```

Comment:
Again white corners here.

Reply:
Deletion of these lines makes regression.